### ISSUE TITLE
TeamCity integration (output version to log)

### ISSUE THREAD

**Issue by slolife** [Zoltu/Zoltu.Versioning#15]
Would love to have TeamCity integration possible:
http://hadihariri.com/2016/08/22/Git-Versioning/#passing-information-to-teamcity

Which would just mean outputting "##teamcity[buildNumber \'${version}\']" to the log

**Comment by MicahZoltu** [comment]
I'm hesitant to couple this plugin that tightly with TeamCity.  For users that aren't using TeamCity, the output log line would just be a nuisance since it isn't needed in their build system.  I could add an option to output it, but I think it would take more `build.gradle` boilerplate to turn the option on then it would to just add this to your log:

``` groovy
ZoltuGitVersioning.enableTeamCityLogLine = true
```

vs

``` groovy
println "##teamcity[buildNumber \'${version}\']"
```

The first option, while perhaps _slightly_ cleaner in a TeamCity `build.gradle`, it would only work with TeamCity and it still requires the TeamCity user to do a little bit of additional TeamCity config.

**Comment by MicahZoltu** [comment]
Oops!  I just realized that this issue was posted in the .NET project, not the similar project I have for Gradle!

That being said, the answer for this project isn't much better for you I think, it is basically the same as the answer I posted over here: https://github.com/Zoltu/Zoltu.Versioning/issues/16#issuecomment-243634190

> This project is very near to being end-of-lifed in favor of GitTools/GitVersion since it is more full featured and their upcoming release will allow for a versioning scheme that matches the scheme this project provides, but with additional features. They call this scheme "Mainline Development" and you can see their docs about it here:
> http://gitversion.readthedocs.io/en/latest/reference/mainline-development/
> A test that shows this working can be seen here:
> https://github.com/GitTools/GitVersion/blob/master/src/GitVersionCore.Tests/IntegrationTests/MainlineDevelopmentMode.cs#L112
> 
> I believe the Mainline Development feature is in beta now, and will be available in their 4.0 release. You may want to check with them to see if they have TeamCity support, or something that makes logging the version easy since I will probably not be committing to this project much more.

**Comment by MicahZoltu** [comment]
Closing this issue as won't fix per the above comment. Sorry!
